packages/app/src/store/run-all-specs-store.ts

```diff
@@ -77,6 +76,8 @@ export const useRunAllSpecsStore = defineStore('runAllSpecs', () => {
     directoryChildrenRef.value = directoryChildren
   }
 
+  const query = useQuery({ query: RunAllSpecsDataDocument, pause: isRunMode || window.__CYPRESS_TESTING_TYPE__ === 'component' })
```

Comment:
Run All Specs is E2E only, drive-by optimization - we don't need to query if we are in CT mode.